src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -959,12 +971,8 @@ boolean isContinueUntilTwoCandidatesRemainEnabled() {
   }
 
   int getNumDeclaredCandidates() {
-    int num = getCandidateCodeList().size();
-    if (!isNullOrBlank(getUndeclaredWriteInLabel())
-        && getCandidateCodeList().contains(getUndeclaredWriteInLabel())) {
-      num--;
-    }
-    return num;
+    // we subtract one for UNDECLARED_WRITE_IN_OUTPUT_LABEL;
+    return getCandidateCodeList().size() - 1;
```

Comment:
I'm unsure why you changed this to no longer check if there's a UWI label? Because now it always has a UWI label internally assigned or something? Is this actually the case even if no UWI label is assigned and "treat blank as UWI" isn't checked?

Reply:
Oh, dang. Good catch. The change I made here was appropriate in an intermediate version of my code, but you're right that it's very wrong now. Will fix.